staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -220,10 +220,16 @@ func (r *crdHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 
 	var handler http.HandlerFunc
+	subresources, err := getCRDSubresourcesForVersion(crd, requestInfo.APIVersion)
+	if err != nil {
+		utilruntime.HandleError(err)
+		http.Error(w, "the server could not properly server the CRD subresources", http.StatusInternalServerError)
```

Comment:
these are not CRD subresources, but CustomResource (CR) subresources. This applies to a number of places.

Reply:
Updated.

nit: the problematic object was the CRD. Maybe we should call it "could not properly serve subresources defined in CRD"

Reply:
"defined by CRDs" is also fine, either way :+1: